src/hooks/core/useCoreChatSystemHook.ts

```diff
@@ -32,14 +33,16 @@ import {getChannelListInfo} from '../../utils/string/StringUtils';
 
 const useCoreChatSystemHook = () => {
   const {localDb, refresh} = useLocalDatabaseHook() as UseLocalDatabaseHook;
-  const {anonProfileId, signedProfileId} = useProfileHook();
+  const {anonProfileId, signedProfileId} = useUserAuthHook();
   const {getAllSignedChannels, getAllAnonymousChannels} = useFetchChannelHook();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The replacement of `useProfileHook` with `useUserAuthHook` is consistent with the changes made in the imports. This should ensure that the newer, preferred hook is used throughout the component.
```diff
- const {anonProfileId, signedProfileId} = useProfileHook();
+ const {anonProfileId, signedProfileId} = useUserAuthHook();
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->